### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by newhinton** [GrapheneOS/os-issue-tracker#3452]

**Comment by MichaelDevon** [comment]
So basically fuzzy search.

**Comment by newhinton** [comment]
Yeah, at least for the last suggestion. The others are more an improvement for the tokenization.

I am generally willing to help out, but i am entirely unfamilliar how the packages from an android fork can be build&tested on their own, so i'd only be able to write some standalone code that could then be integrated in the matcher utility.
